pr_agent/git_providers/github_provider.py

```diff
@@ -112,6 +112,11 @@ def get_files(self):
                 self.git_files = self.pr.get_files()
             return self.git_files
 
+    def get_num_of_files(self):
+        if self.git_files:
+            return self.git_files.totalCount
+        else:
+            return -1
```

Comment:
To ensure `get_num_of_files` returns accurate information at all times, consider initializing `git_files` within this method if it's not already initialized. This ensures that `totalCount` is always available for use. [important]